### PR TITLE
Update release_notes.md

### PR DESCRIPTION
**Pull request by lizygeogy**
Updated releasenotes for 3.10

### CODE REVIEW

File: docs/release_notes.md [hunk 1]
```diff
@@ -1,5 +1,211 @@
...
+
+- Multiple redirections occur when logging into drools workbench through proxy URL. (SPOI-13254)
+>**Workaround** : Click the **Back** button on the browser to go to the Drools WorkBench login page. You can also close the current tab and click the Proxy URL once again to go to the login page.
+
+- Transient services are not deleted when the corresponding app is terminated. (SPOI-12983)
+- Schema Aware operators should not perform validations in setter methods. (SPOI-13095)
+- Proxy URL for newly created Superset service (from UI) does not work. (SPOI-13454)
+- GATEWAY\_CONNECT\_ADDRESS parameter is misspelt as GATEWAY\_CONNECT\_ADDRESSS. (SPOI-13449)
+- Fraud App: Physical DAG is broken (SPOI-13388)
+- Launching the configuration when the services are not installed gives the following error:
```
Comment (wvary) [inline docs/release_notes.md]: @lizygeogy Why is this in the release notes if it cannot be reproduced and the Jira is closed?

File: docs/release_notes.md [hunk 2]
```diff
@@ -1,5 +1,211 @@
...
+
+- _KafkaExactlyOnceOutputOperator_ does not work with Kerberised cluster. (SPOI-13182)
+- Operator in fraud-prevention App dies while attempting to record a sample. (SPOI-13194)
+- The java script validation cannot terminate a java script, if it is running longer than the timeout period. (SPOI-12265)
+- Application does not shutdown even after consuming the end offset in Kafka. ( SPOI-12955)
+- OAS is unable to handle multiple intervals in select query. (SPOI-12865)
+- DT services with secure docker setup does not work. (SPOI-13104)
+>**Workaround:** Docker in SSL mode requires mutual TLS (MTLS) which is not supported. Make sure your docker daemon uses either UNIX sockets or plain TCP sockets
+>
+- While upgrading the RTS from 3.9.2 to 3.10.0, the property dt.gateway.artifactHubLocation in dt-site.xml  is not getting set. (SPOI-13342)
```
Comment (wvary) [inline docs/release_notes.md]: @lizygeogy I thought we were going to remove this as an issue.  Did you ever hear back from @sanjaypujare or @rohmurthy ?

File: docs/release_notes.md (hunk 1 above)
Comment (lizygeogy) [reply docs/release_notes.md]: @wvary Please specify the defect number. I am unable to understand which one you are referring to.

File: docs/release_notes.md [hunk 3]
```diff
@@ -1,5 +1,211 @@
...
+- [DataTorrent's RTS Apoxi ™ framework with added features](#apoxi)
+- [New and enhanced AppFactory applications](#appfactory)
+- [Miscellaneous features](#miscellaneous)
+
+## RTS Platform <a name="RTSPlatform"></a>
+
+The following RTS Platform features are included in this release:
+
+- [Support for OLAP with Druid](#olapwithdruid)
+- [Expanded Support for Machine Learning and AI](machinelearning)
```
Comment (wvary) [inline docs/release_notes.md]: @lizygeogy Is this link supposed to go to another page or the anchor tag below?

File: docs/release_notes.md [hunk 4]
```diff
@@ -1,5 +1,211 @@
...
+
+The following features of RTS Apoxi framework are included in 3.10 release:
+
+- [Store and Replay](#storeandreplay)
+- [Drools Workbench](#drools)
+- [Application Backplane](#appbackplane)
+- [Azure HDI Deployment](#azure)
+- [New iFrame Widget](#iframe)
+- [Configuration Artifacts for Schemas and Rules](#configartifacts)
+- [Enhanced RTS UI for Launch and Application Configuration](launchandappconfig)
```
Comment (wvary) [inline docs/release_notes.md]: @lizygeogy Is this link correct?

File: docs/release_notes.md [hunk 5]
```diff
@@ -1,5 +1,211 @@
...
+## AppFactory <a name="appfactory"></a>
+
+The AppFactory includes an extensive library of popular open-source and proprietary operators for data input/output, enrichment, analytical functions, and more. These operators are further combined into pre-tested, streaming micro data services for common patterns such as data ingestion, real-time synchronization, anomaly detection, machine scoring, etc.
+
+The following features are included in AppFactory for 3.10 release:
+
+- [Omni-Channel Fraud Prevention Application V2](#fpa)
+- [Account Takeover Prevention Application](#ato)
+- [New Use Case: Retail Recommender PoC](#retail)
+- [Micro Data Service (MDS): Online Analytics Service](oas)
```
Comment (wvary) [inline docs/release_notes.md]: @lizygeogy Is this link correct?

**Comment by wvary** [comment]
@lizygeogy Can you change occurrences of anchor tags from after the h3 to before.
For example change this:
`### Online Analytics Service (OAS)<a name="oas"></a>`

To:
`<a name="oas"></a>`
`### Online Analytics Service (OAS)`

Please make sure this is done on every anchor tags of the new text.

File: docs/release_notes.md [hunk 6]
```diff
@@ -1,5 +1,211 @@
-DataTorrent RTS Release Notes
+DataTorrent Real-time Streaming (RTS) Release Notes
 ========================================================================================================================
+Version: 3.10
+------------------------------------------------------------------------------------------------------------------------
+
+Companies want to gain quick insights and take action on impacting events at the speed of their businesses. DataTorrent's Real-time Streaming (RTS) platform makes it easy to deliver fast data analytics applications that drive business outcomes using the latest innovation in data science and machine learning. It integrates best-of-breed open source technology innovation to provide all the features that a business needs to develop and deliver best-in-class, fast data applications.
+
+DataTorrent RTS 3.10 version includes the following newfeatures, resolved issues, and known issues.
```
Comment (peyao) [inline docs/release_notes.md]: typo: newfeatures needs a space

File: docs/release_notes.md [hunk 7]
```diff
@@ -1,5 +1,211 @@
...
+## RTS Platform <a name="RTSPlatform"></a>
+
+The following RTS Platform features are included in this release:
+
+- [Support for OLAP with Druid](#olapwithdruid)
+- [Expanded Support for Machine Learning and AI](machinelearning)
+)
+### Support for Real-time OLAP with Druid <a name="olapwithdruid"></a>
+
+Provides customers with the ability to slice and dice data in real-time to get the information needed to compute and compare it to historical trends. This capability is delivered as a pre-built component that tightly integrates Apex, Druid, and SuperSet for real-time plus historical dashboards and visualizations.   Druid is an open-source project that is hardened and delivered as part of the RTS platform so that a separate, dedicated Druid cluster is not required.
```
Comment (peyao) [inline docs/release_notes.md]: Can we write SuperSet as Superset

**Comment by sashadt** [comment]
@wvary I've replaced the tags as well.

**Pull request merged**